### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#309] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181403](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/736e1ef7-3561-4cd5-8387-8a358c06915b)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/a73b69e7-82ff-479d-a897-87be08c53a2e)

**Comment by daromacarlo** [comment]
Qualcuno sa dirmi se è corretto?


![Immagine 2023-12-12 223613](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b997ecf9-ad40-49a0-bcf4-144b665d284a)
![Immagine 2023-12-12 223632](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/ee60f57e-7b0d-47ee-a77b-7832530f7a59)
![Immagine 2023-12-12 223648](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/96bddc82-d15b-45eb-be6e-12cc2fbb90ac)
